core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -255,6 +245,38 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     }
 
     ngOnInit(): void {
+        this.store.code$
+            .pipe(
+                takeUntil(this.destroy$),
+                switchMap((code) => {
+                    return this.store.clientHost$.pipe(
+                        map((clientHost) => {
+                            return { code, clientHost };
+                        })
+                    );
+                })
+            )
```

Comment:
@KevinDavilaDotCMS You don't need to check the `clientHost` in the `code$` selector because the `page.rendered` only comes in `VTL` pages. You can use a filter when "rendered" is null not to call the function.

Reply:
I dont use a selector with both vars because it have a strange behaviour (i remember a selector that returns a single value acts different than a selector that returns a object, but i cannot find the doc issue)
But the filter of `rendered` proposed by rafa works, i added it